test/raygun.messageBuilder_test.js

```diff
@@ -404,7 +405,7 @@ test("custom tags", function (t) {
     builder.setTags(["a", "bb", "c"]);
     var message = builder.build();
 
-    tt.deepEqual(message.details.tags, ["a", "bb", "c"]);
```

Comment:
`deepEqual()` is not part of `tap`, replaced by the `deepEqual` from the `assert` module.